jb/itch5/array_based_order_book.hpp

```diff
@@ -0,0 +1,617 @@
+#ifndef jb_itch5_array_based_order_book_hpp
+#define jb_itch5_array_based_order_book_hpp
+
+#include <jb/itch5/price_field.hpp>
+#include <jb/itch5/price_levels.hpp>
+#include <jb/config_object.hpp>
+#include <jb/feed_error.hpp>
+#include <jb/log.hpp>
+
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <map>
+#include <sstream>
+#include <utility>
+#include <vector>
+
+namespace jb {
+namespace itch5 {
+
+/// A simple representation for price + quantity
+using half_quote = std::pair<price4_t, int>;
```

Comment:
renamed to quote_defaults.hpp to add 4 static members (empty_bid, empty_offer, empty_bid_price, empty_offer_price)